xapian-applications/omega/clickmodel/simplifieddbn.cc

```diff
@@ -73,9 +75,9 @@ getClicksList(vector<vector<string>> &sessions)
 		clickstring += sessions[i][CLICKS][j];
 
 		// Get click count string.
-		string delimiter = ":";
-		int delimiterPos = clickstring.find(delimiter);
-		clickcount = clickstring.substr(delimiterPos + 1, clickstring.length() - 1 - delimiterPos);
+		int delimiter_pos = clickstring.find(':');
+		clickcount = clickstring.substr(delimiter_pos + 1,
+			     clickstring.length() - 1 - delimiter_pos);
```

Comment:
If you omit the second argument of `substr()`, it defaults to `string::npos` which means "the rest of the string":

http://en.cppreference.com/w/cpp/string/basic_string/substr